src/coreclr/jit/emit.cpp

```diff
@@ -4068,6 +4080,7 @@ void emitter::emitDispIG(insGroup* ig, bool displayFunc, bool displayInstruction
 
         printf("\n");
 
+#if !defined(TARGET_RISCV64)
```

Comment:
I put a TODO-RISCV64-Bug

Reply:
Does this mean `DOTNET_JitDisasm` does not work for RISC-V?

Reply:
Ping @clamp03 on the above question.

If `DOTNET_JitDisasm` does not work yet, is it in the plans to work on it to make it work?

Reply:
Actually, we don't any plan for `DOTNET_JitDisasm`. Currently, I and my colleagues are focusing on fixing bugs in coreclr tests using `DOTNET_JitDump`, `GDB`, and others. Using the tools, we can improve a pass rate of `CodeGenBringUp` tests from around 40% to 90%. However, if you believe that `DOTNET_JitDisasm` is a significant feature, we can prioritize it accordingly. And we can implement internally and then push a new PR for `DOTNET_JitDisasm` after this patch is merged.
Thank you.

Reply:
Thanks. So if I understand correctly, you are using GDB to view the code produced by the JIT?

We are ok with delaying disassembly support to a future PR, but we do think it is important for all our backends to support outputting the disassembly as part of JitDisasm and JitDump. In particular to allow JIT developers and other contributors to more easily pick up and work on RISC-V themselves. However, I will leave the prioritization of it up to you.

On that same topic it would be great if you at some point could open a "road map" issue to track some of these missing bits for RISC-V and how you are prioritizing them.

Reply:
`DOTNET_JitDump` is working although it is not perfectly.
We will open an issue for "road map" and "prioritization" after discussion.
Thank you!